data_steward/bq_utils.py

```diff
@@ -221,18 +219,21 @@ def load_cdm_csv(hpo_id,
     :param cdm_table_name: name of the CDM table
     :return: an object describing the associated bigquery job
     """
+    app_id: str = app_identity.get_application_id()
+    storage_client = StorageClient(app_id)
+    hpo_bucket = storage_client.get_hpo_bucket(hpo_id)
+
     if cdm_table_name not in resources.CDM_TABLES:
         raise ValueError(
             '{} is not a valid table to load'.format(cdm_table_name))
 
-    app_id = app_identity.get_application_id()
-    if dataset_id is None:
-        dataset_id = get_dataset_id()
-    bucket = gcs_utils.get_hpo_bucket(hpo_id)
-    gcs_object_path = 'gs://%s/%s%s.csv' % (bucket, source_folder_prefix,
-                                            cdm_table_name)
+    if not dataset_id:
+        dataset_id: str = get_dataset_id()
+
+    gcs_object_path: str = 'gs://%s/%s%s.csv' % (
+        hpo_bucket.name, source_folder_prefix, cdm_table_name)
```

Comment:
This could be an f-string

Reply:
Thank you for pointing these out.  Personally, to me, they are the better syntax (the worst to me is the `"%s" % foo`)